Cargo.toml

```diff
@@ -13,6 +13,9 @@ build = "build.rs"
 readme = "README.md"
 keywords = ["sandbox", "FreeBSD", "capsicum"]
 
+[package]
```

Comment:
Eliminate duplicate line.
```suggestion
```
Also, while this would work in general I prefer to whitelist rather than blacklist.  Like this:
```toml
include = ["src/**/*", "test/**/*", "LICENSE", "README.md", "CHANGELOG.md"]
```

Reply:
Good point! I had forgotten to exclude the rustfmt.tom... A list like this avoids errors like that.